src/main/java/org/colonelkai/publictransit/line/LineDirection.java

```diff
@@ -16,6 +16,10 @@ public enum LineDirection {
         this.isPositive = isPositive;
     }
 
+    public boolean isPositive() {
+        return this.isPositive;
+    }
+
     private int getAddition(boolean isReverse) {
         int addition = this.isPositive ? 1 : -1;
```

Comment:
I got a question about this. If we already know if this is positive or negative, whats the need for it to be reversed?

Reply:
When the line isnt one way and therefore your going reverse, the calculation need to be adjusted. 

While we could swap the enum in that case the only sane way to check that is to use boolean logic hence why i have exposed if its positive. 

I could probably clean up this class a bit more, removing the need for the isReverse in the methods, but I find it cleaner to pass in isReverse rather then switching the enum outside the class